svgo.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    multipass: true,
+    plugins: [
+      {
+        name: 'preset-default',
+        params: {
+          overrides: {
+            removeTitle: false,
+            removeViewBox: false,
+          },
+        },
+      },
+      {
+        name: 'removeAttributesBySelector',
+        params: {
+          selectors: [
+            {
+              selector: '[preserveAspectRatio="none"]',
+              attributes: 'preserveAspectRatio',
+            },
+            {
+              selector: 'svg',
+              attributes: 'style',
+            },
+          ],
+        },
+      },
+    ],
+  };
```

Comment:
```suggestion
  };
  
```
it need an empty line at the end, please check your prettier and see whay this is not added on save